components/clearbutton/stories/clearbutton.mdx

```diff
@@ -0,0 +1,61 @@
+import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';
+
+import * as ClearButton from './clearbutton.stories';
+
+<Meta of={ClearButton} title="Docs" />
+
+<Title of={ClearButton} />
+<Description of={ClearButton} />
+
+### Usage Notes
+
+Use the correct cross icon size that corresponds to the t-shirt size you require for the clear button.
+
+<table>
```

Comment:
Am I doing something wacky- I could not for the life of me get a markdown table to work in this file. Is this not how you'd do a md table? 

```
| T-Shirt Size  | Icon Size |
| ------------- | --------- |
| spectrum-ClearButton--sizeS | spectrum-css-icon-Cross75 |
| spectrum-ClearButton--sizeM | spectrum-css-icon-Cross100 |
| spectrum-ClearButton--sizeL | spectrum-css-icon-Cross200 |
| spectrum-ClearButton--sizeXL | spectrum-css-icon-Cross300 |
```

It even works in this comment!! 🥲
| T-Shirt Size  | Icon Size |
| ------------- | --------- |
| spectrum-ClearButton--sizeS | spectrum-css-icon-Cross75 |
| spectrum-ClearButton--sizeM | spectrum-css-icon-Cross100 |
| spectrum-ClearButton--sizeL | spectrum-css-icon-Cross200 |
| spectrum-ClearButton--sizeXL | spectrum-css-icon-Cross300 |

I kept getting an unformatted table:

<img width="1013" alt="Screenshot 2024-05-29 at 10 34 03 AM" src="https://github.com/adobe/spectrum-css/assets/69602589/fc1cc1e2-6620-446f-a6d6-ab3f033cd60d">

Reply:
Weird! I wonder if it is a Storybook limitiation--I will test this out.

Reply:
I took a 20-minute detour trying to recall/recreate this in vain because it sounded like something I encountered when I was working on corner rounding--pretty sure you're correct and markdown table syntax is wonky in Storybook.

I want to say I _think_ you can still do it if you want to and I had some luck creating `.md` files that I imported into the `.mdx` file? The only evidence I can find of that though is that we have [these weird markdown table files that I don't think we're using](https://github.com/adobe/spectrum-css/blob/spectrum-two/.storybook/foundations/corner-rounding/component-size-tokens-medium.md) but I think that might be a remnant of that experiment 🤔